mappings/net/minecraft/entity/EntityLiving.mapping

```diff
@@ -43,6 +48,13 @@ CLASS none/sp net/minecraft/entity/EntityLiving
 		ARG 1 stack
 	METHOD a getAttributeInstance (Lnone/tb;)Lnone/tc;
 		ARG 0 attribute
+	METHOD a dropEquipment (ZI)V
+		ARG 0 recentlyHitByPlayer
```

Comment:
Oh. I keep forgetting the signature is actually in the Enigma file, as it's hard to miss the binary signature when there's no classes in it.

I figured it was an EntityPlayer argument :L